src/discretize.jl

```diff
@@ -41,12 +50,12 @@ function discretize(x::Vector, Flow::Union{BouncyParticle, Boomerang}, dt0)
     _, ξ, θ = x[k]
     τ = x[k+1][1]
     clock = 0.0
-    Ω = trajectory([clock => ξ])
+    Ω = trajectory([clock => deepcopy(ξ)])
```

Comment:
Here

Reply:
This is an old version that would not work for the sticky Boomerang sampler. The current way to discretize the path is impelemnted in src/trace.jl (which probably will fail again for discretizing the path of the sticky Boomerang sampler). I think a `deepcopy` or at least a `copy` is necessary here.